webclient/app/src/app/commons/entityCard/EntityCard.jsx

```diff
@@ -24,39 +24,43 @@ function EntityCard(props) {
 
     function renderAttributes(entity) {
 
-        return entity.fields.map((field, index) => {
+        if (!entity.fields || entity.fields.length === 0) {
             return (
-
-                <TableRow key={index}>
-                    <TableCell className={entityCardStyles.relationshipPlaceholder}>
-                        <div className={"anchor_" + entity.name + "_" + field.fieldName + "_l"}/>
-                    </TableCell>
-                    <TableCell>{field.fieldName}</TableCell>
-                    <TableCell>{field.fieldType}</TableCell>
-                    <TableCell className={entityCardStyles.relationshipPlaceholder}>
-                        <div className={"anchor_" + entity.name + "_" + field.fieldName + "_r"}/>
+                <TableRow key={0}>
+                    <TableCell colSpan={4} >
+                        <div className={entityCardStyles.statementWrapper}>
+                            <Statement message={t("entityCard.noFields")}/>
+                        </div>
                     </TableCell>
                 </TableRow>
             )
-        })
+        } else {
+            return entity.fields.map((field, index) => {
+                    return (
+                    <TableRow key={index}>
+                        <TableCell />
+                        <TableCell>{field.fieldName}</TableCell>
+                        <TableCell>{field.fieldType}</TableCell>
+                        <TableCell />
+                    </TableRow>
+                )
+            })
+        }
     }
 
     function renderFieldsTable(entity) {
-        if (!entity.fields || entity.fields.length === 0) {
```

Comment:
Why do you want to print the header, even if there is no content?

Reply:
I need a places to add the relationship anchors even if there is no field defined